src/App.tsx

```diff
@@ -5,7 +5,17 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const handleAddMovie = (movie: Movie) => {
+    setMovies((prevMovies: Movie[]) => {
```

Comment:
TS already knows about it
```suggestion
    setMovies((prevMovies) => {
```